babel.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function(api) {
+  api.cache.never();
+  return {
+    presets: ['@babel/preset-env'],
+    plugins: ['babel-plugin-transform-custom-element-classes'],
```

Comment:
my understanding is that we don't need this for Babel 7, but is that incorrect? I thought this plugin was only for Babel 6.

Reply:
Maybe? That is something that can be tested.

Reply:
Yep good catch, `babel-plugin-transform-custom-element-classes` has been removed